### DISCUSSION TITLE
How to insert logos at the bottom of the website page with quarto?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#6007] [Q&A] [answered]
### Description

How to insert 2 logos at the bottom of the website page, one to the right and one to the left?

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/websites/website-navigation.html#page-footer>.

**Comment by ManuelSpinola** [comment]
Thank you very much. It works.

El jue, 22 jun 2023 a las 7:34, Mickaël Canouil ***@***.***>)
escribió:

> See https://quarto.org/docs/websites/website-navigation.html#page-footer.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/6007#discussioncomment-6252267>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB2JQXLP5ISRNP7IZA3XMRCVZANCNFSM6AAAAAAZQGYS7I>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Sitio web institucional: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Sitio web personal: Sitio personal <https://mspinola-sitioweb.netlify.app>
Blog sobre Ciencia de Datos: Blog de Ciencia de Datos
<https://mspinola-ciencia-de-datos.netlify.app>
